netlify.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+  command = "pnpm build"
+  publish = ".next"
+
+[[plugins]]
+  package = "@netlify/plugin-nextjs"
```

Comment:
Please, remove the plugins section. This is setup automatically by Netlify for Next.js projects.